easybuild/easyconfigs/g/Gctf/Gctf-wrapper.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Wrap around Gctf to load the required toolchain so it can use the
+# expected version of CUDA.
```

Comment:
@akesandgren Comment should be fixed, we're only loading the required CUDA, not any toolchain?